reference/core/ingress-controller.md

```diff
@@ -1,88 +1,16 @@
 # Ambassador as an Ingress Controller
 
-Starting with version 0.80.0, Ambassador can act as a Kubernetes
-[ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/),
-reading configuration data from Kubernetes 
-[`Ingress`](https://kubernetes.io/docs/concepts/services-networking/ingress/) resources.
-This makes it easier to work with other `Ingress`-oriented tools within the Kubernetes
-ecosystem, and it makes it easier for users migrating from other
-ingress controllers to try Ambassador.
+An `Ingress` resource is a popular way to expose Kubernetes services to the Internet. In order to use `Ingress` resources, you need to install an [ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/). Ambassador can function as a full-fledged Ingress controller, making it easy to work with other `Ingress`-oriented tools within the Kubernetes ecosystem.
 
 ## When and How to Use the `Ingress` Resource
 
-In order to use the `Ingress` resource effectively, it's important to understand not
-just how to use it, but also when to use it, and how it interacts with CRDs and
-annotations.
+If you're new to Ambassador and Kubernetes, we'd recommend you start with our [quickstart](/user-guide/getting-started/).
```

Comment:
I wonder if it's worth extending this a bit. "...we'd recommend you start with our quickstart, instead of using Ingress." ?